cli.py

```diff
@@ -112,4 +112,5 @@ async def messaging(
 
     message_dict = {i: loaded_message[i]["value"] for i in loaded_message}
     message = topic.MessageContents(**message_dict)
+    await topic.topic.maybe_declare()
```

Comment:
how long does it wait? what happens if the topic does not come up? shall we move topic creation to the init-job?

Reply:
It's a very short wait, if the topic doesn't exist yet it creates it.  I think it's best to put here as this should be a general purpose CLI, if we don't add this line the bug will persist.  Adding it to the init job feels like an unnecessary work-around especially because we will have to add to it every time a new topic is created.